tjreports/site/models/report.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @package    Com_Tmt
+ * @copyright  Copyright (C) 2009 -2015 Techjoomla, Tekdi Web Solutions . All rights reserved.
+ * @license    GNU GPLv2 <http://www.gnu.org/licenses/old-licenses/gpl-2.0.html>
+ * @link       http://www.techjoomla.com
+ */
+
+// No direct access.
+defined('_JEXEC') or die;
+
+/**
+ * Tmt model.
+ *
+ * @since  1.0
+ */
+class  TjreportsModelReport extends JModelAdmin
+{
+	/**
+	 * Method to get a table object, load it if necessary.
+	 *
+	 * @param   string  $type    The table name. Optional.
+	 * @param   string  $prefix  The class prefix. Optional.
+	 * @param   array   $config  Configuration array for model. Optional.
+	 *
+	 * @return  JTable  A JTable object
+	 *
+	 * @since   1.6
+	 */
+
+	public function getTable($type = 'Tjreport', $prefix = 'TjreportsTable', $config = array())
+	{
+		return JTable::getInstance($type, $prefix, $config);
+	}
+
+	/**
+	 * Method to save the form data.
+	 *
+	 * @param   array  $data  The form data.
+	 *
+	 * @return  mixed  The user id on success, false on failure.
+	 *
+	 * @since  1.0
+	 */
+	public function save($data)
```

Comment:
@hemantpurohit1  If you are doing anything special in this save function, no need to write it